example/src/components/ImagePickerExamples.js

```diff
@@ -2,20 +2,68 @@
 
 import React from 'react';
 import ImagePicker from '../../../src/components/ImagePicker';
+import TextInput from '../../../src/components/TextInput';
 import Code from '../../../src/components/Code';
 
-export default function ImagePickerExamples() {
-  return (
-    <section className="_full-width-row" id="ImagePicker">
-      <div className="_container _container_large">
-        <h2 className="base--h2">ImagePicker</h2>
-        <ImagePicker />
-        <Code language="html">
+export default React.createClass({
+
+  getInitialState() {
+    return {
+      urlError: null,
+      fileError: null,
+      error: null,
+    };
+  },
+
+  render() {
+    return (
+      <section className="_full-width-row" id="ImagePicker">
+        <div className="_container _container_large">
+          <h2 className="base--h2">ImagePicker</h2>
+          <ImagePicker />
+          <Code language="html">
 {'<ImagePicker />'}
```

Comment:
I think the example should probably include enough props to make it useful, so maybe `onClickTile`, `onDrop`, and `onUrlInputChange` with each just doing a console.log.